src/main/java/step1/domain/StringAddCalculator.java

```diff
@@ -0,0 +1,55 @@
+package step1.domain;
+
+import step1.exceptin.StringAddCaculatorException;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+    private static final int ZERO = 0;
+    private static final String DEFAULT_SEPARATOR = ",|:";
+    private static final String CUSTOM_SEPARATOR = "//(.)\n(.*)";
+
+    public static int splitAndSum(String input) {
+        int result;
+        if (input == null || input.isEmpty()) {
+            return ZERO;
+        }
+        String[] tokens = splite(input);
+        result = sum(tokens);
+        return result;
```

Comment:
```suggestion
        if (input == null || input.isEmpty()) {
            return ZERO;
        }
        String[] tokens = splite(input);
        return sum(tokens);
```

이렇게 표현하면 result 변수를 사용하지 않아도 되겠어요! 😄